README.md

```diff
@@ -8,21 +8,209 @@
+
+4. After the installation, you will be redirected to the localhost server you started in the previous step. The server will print the authorization code in the console.
+
+   ```json
+   {
+     "token_type": "bearer",
+     "refresh_token": "<Refresh Token>",
+     "access_token": "<Access Token>",
+     "expires_in": 1800
+   }
+   ```
+
+5. Store the `refresh_token` securely for use in your application
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the `HubSpot Deals` connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `hubspot.crm.obj.deals` module.
+
+```
+import ballerinax/hubspot.crm.obj.deals;
+```
+
+### Step 2: Instantiate a new connector
+
+1. Create a `deals:ConnectionConfig` with the obtained access token and initialize the connector with it.
+
+   ```ballerina
+   configurable string clientId = ?;
+   configurable string clientSecret = ?;
+   configurable string refreshToken = ?;
+   
+   deals:OAuth2RefreshTokenGrantConfig auth = {
+        clientId,
+        clientSecret,
+        refreshToken,
+        credentialBearer: oauth2:POST_BODY_BEARER
+    };
+   final deals:Client hubSpotDeals = check new ({ auth });
+   ```
+
+2. Create a `config.toml` file and, configure the obtained credentials in the above steps as follows:
+   ```toml
+   clientId = "<Client ID>"
+   clientSecret = "<Client Secret>"
+   refreshToken = "<Access Token>"
+   ```
+
+### Step 3: Use Connector Operations
+
+Utilize the connector's operations to create, update and delete deals etc.
+#### Create a Deal
+```ballerina
+   deals:SimplePublicObjectInputForCreate payload = {
+        properties: {
+            "pipeline": "default",
+            "dealname": "Test Deal",
+            "amount": "100000"
+        }
+    };
+
+    SimplePublicObject out = check hubSpotDeals->/.post(payload = payload);
+```
+#### List Deals
+```ballerina
+   deals:CollectionResponseSimplePublicObjectWithAssociationsForwardPaging dealsList = check  hubSpotDeals->/;
+```
 
 ## Examples
 
-The `Ballerina HubSpot CRM Deals Connector` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-hubspot.crm.object.deals/tree/main/examples/), covering the following use cases:
+The `ballerinax/hubspot.crm.obj.deals` connector provides practical examples illustrating usage in various scenarios.Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.deals/tree/main/examples), covering the following use cases:
+
+1. [Manage Deals](https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.deals/tree/main/examples/sales-pipeline) - see how the Hubspot API can be used to create deal and manage it through the sales pipeline.
```

Comment:
Is this description correct ? The example title is `Manage Deals` but the description is bit different.